source/support_contact_and_more_information/index.html.md.erb

```diff
@@ -9,45 +9,32 @@ weight: 220
 
 You can get notifications about GOV.UK Pay’s status to your email, phone, RSS reader or website, by subscribing on our [status page](https://payments.statuspage.io/#).
 
-## Incident severity
+After you contact us, the GOV.UK Pay team will contact you using the Zendesk ticketing system.
```

Comment:
Should we include a general intro here to our support process? e.g. Our office hours are 9.30am to 5.30pm, Monday to Friday. You can email us at govuk-pay-support@digital.cabinet-office.gov.uk. We’ll reply within 2 working days. The team are available out of hours in an emergency. Otherwise I wonder if this support page feels a bt like it just talks about incidents?

Reply:
Ah good point - I can add a bit of this to the '## General feedback' section, as that's where we're meeting this need. Do we reply to general questions and feedback within 2 days?

Reply:
Yes, 2 working days for general questions. 1 working day response for go live requests.

Reply:
@miriamalice Thanks - now added in https://github.com/alphagov/pay-tech-docs/pull/540/commits/b619876347215fad393db9fb270068475b0551a1